devel/build

```
@@ -19,7 +19,14 @@ export GIT_REVISION=$(git describe --tags --abbrev=40 --always --dirty || true)
 # Calling `docker run nextstrain/base` will still only pull down the small base
 # image rather than pulling down the larger nextstrain/base-builder image.
 
-docker build \
+platform=linux/amd64
+
+# `buildx create` is necessary to use a driver that supports multi-platform
+# images.
+builder=$(docker buildx create --use)
+
+docker buildx build \
+    --platform $platform \
```

…and correspondingly, we should pass the name to `--builder` here.

```suggestion
docker buildx build \
    --builder "$builder" \
    --platform $platform \
```